t/12_sles4sap_publicccloud.t

```diff
@@ -3,11 +3,12 @@ use warnings;
 use Test::MockModule;
 use Test::Exception;
 use Test::More;
+use Test::Mock::Time;
```

Comment:
This is the IMPORTANT line of the review: https://metacpan.org/pod/Test::MockTime

Reply:
You're not setting the time anywhere. So what is this expected to do by default?

Reply:
Well, I saw it was just merged... I was just wondering since I don't see anything in the docs how this makes things faster

Reply:
> So what is this expected to do by default?

To replace functions like `sleep` so it doesn't actually sleep.

Reply:
@mpagot  you posted the link to different module documentation which is pretty confusing